src/main/resources/openapi/audio.yaml

```diff
@@ -29,6 +29,31 @@ paths:
             schema:
               $ref: '#/components/schemas/AudioRequestDetails'
         description: Audio Request item to add
+    delete:
```

Comment:
I think this endpoint should go along the new one I added this sprint
[audio-requests.yaml](https://github.com/hmcts/darts-api/blob/master/src/main/resources/openapi/audio-requests.yaml)

DELETE `/audio-requests/{id}` ?

FYI @hemantasharma1129

Reply:
It should be part of AudioController which has endpoints for requesting an Audio, downloading an Requested Audio etc. AudioRequestController may no longer be required? We can discuss about it.